data/names/ru.json

```diff
@@ -3193,70 +3221,93 @@
       "Бентли",
       "Бентон",
       "Бенуа",
+      "Берг",
       "Бергер",
       "Бергман",
-      "Берг",
+      "Берд",
       "Берден",
       "Бердетт",
-      "Берд",
+      "Бердик",
       "Бержерон",
+      "Берк",
+      "Беркетт",
+      "Беркс",
+      "Беркхарт",
       "Берман",
       "Бермудес",
       "Берналь",
+      "Бернар",
       "Бернард",
       "Бернетт",
+      "Бернс",
       "Бернштейн",
+      "Бернье",
+      "Берр",
       "Берри",
+      "Беррис",
       "Берроуз",
+      "Берт",
+      "Бертон",
       "Бертран",
       "Берч",
       "Бесерра",
       "Бест",
       "Бетанкур",
       "Беттс",
       "Биб",
-      "Биверс",
+      "Биби",
       "Бивер",
+      "Биверс",
       "Биггс",
       "Бигелоу",
-      "Биллингсли",
-      "Биллингс",
+      "Бизли",
       "Бил",
+      "Биллингс",
+      "Биллингсли",
       "Бим",
+      "Бин",
       "Бингем",
       "Бинум",
-      "Бин",
       "Бирд",
       "Бирн",
+      "Бирч",
       "Бисли",
       "Бити",
       "Битти",
       "Бич",
       "Бишоп",
+      "Бланд",
+      "Бланк",
       "Бланкеншип",
       "Бланко",
-      "Бланк",
       "Блант",
       "Бланшар",
+      "Блаунт",
       "Блевинс",
       "Бледсо",
-      "Блейкли",
       "Блейк",
+      "Блейкли",
+      "Блек",
+      "Блидсо",
       "Блисс",
       "Блок",
       "Блум",
+      "Блэк",
+      "Блэкберн",
       "Блэкбёрн",
       "Блэквелл",
       "Блэкман",
       "Блэкмон",
-      "Блэк",
       "Блэлок",
       "Блэнд",
       "Блэнтон",
       "Блэр",
-      "Блюм",
       "Блю",
+      "Блюм",
+      "Бо",
+      "бобы",
       "Боггс",
+      "Боевой",
```

Comment:
This is a translation of an adjective that means either "military" or "combat" or "battle" (as in, either feeling like either of those or belonging to one of those). Presumably, [Battle](https://en.wikipedia.org/wiki/Battle_(surname\)) is the original. It would be transcribed Баттл.